.github/workflows/validations.yaml

```diff
@@ -7,7 +7,8 @@ on:
   pull_request:
 
 env:
-  GO_VERSION: "1.17.x"
+  GO_VERSION: "1.18.x"
```

Comment:
we could run all of static-analysis under 1.17, which would minimize changes

Reply:
done